elastalert/elastalert.py

```diff
@@ -1129,6 +1134,12 @@ def run_all_rules(self):
                 elastalert_logger.info("Ran %s from %s to %s: %s query hits (%s already seen), %s matches,"
                                        " %s alerts sent" % (rule['name'], old_starttime, pretty_ts(endtime, rule.get('use_local_time')),
                                                             total_hits, self.num_dupes, num_matches, self.alerts_sent))
+
+                self.statsd.gauge('query.hits', total_hits)
```

Comment:
this is still missing the label of the rule name and the time_taken metric

Reply:
added rule names and time in seconds metric